source/features/avoid-accidental-submissions.tsx

```diff
@@ -20,19 +20,20 @@ function onKeyDown(event: delegate.Event<KeyboardEvent, HTMLInputElement>): void
 	}
 
 	event.preventDefault();
-	select('#issue_body, #pull_request_body, #commit-description-textarea', event.delegateTarget.form!)!.focus();
+	select('#issue_body, #pull_request_body, #commit-description-textarea, #merge_message_field', event.delegateTarget.form!)!.focus();
```

Comment:
Can you split both selector on multiple lines? e.g.
```
	select([
		'#issue_body', 
		#pull_request_body, 
		#commit-description-textarea, 
		#merge_message_field'
	], event.delegateTarget.form!)!.focus();
```

Delegate will need a .join() at the end of the array, but `select` doesn't

Reply:
Ok, done.